src/Committee/EventListener/DeclinePendingCandidacyInvitationsListener.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Committee\EventListener;
+
+use App\Committee\Election\CandidacyManager;
+use App\Entity\CommitteeCandidacyInvitation;
+use App\Repository\CommitteeCandidacyInvitationRepository;
+use App\TerritorialCouncil\Events;
+use App\VotingPlatform\Event\CandidacyInvitationEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class DeclinePendingCandidacyInvitationsListener implements EventSubscriberInterface
+{
+    private $repository;
+    private $candidacyManager;
+
+    public function __construct(CandidacyManager $candidacyManager, CommitteeCandidacyInvitationRepository $repository)
+    {
+        $this->candidacyManager = $candidacyManager;
+        $this->repository = $repository;
+    }
+
+    public static function getSubscribedEvents()
+    {
+        return [
+            Events::CANDIDACY_INVITATION_ACCEPT => 'onCandidacyInvitationAccept',
```

Comment:
Shouldn't be `Events::CANDIDACY_INVITATION_DECLINE` instead? ... and also `onCandidacyInvitationPending`

Reply:
no, here we decline all others pending invitations when the user accept an invitation.